DuckDuckGo/Subscription/Views/SubscriptionSettingsView.swift

```diff
@@ -31,7 +31,7 @@ struct SubscriptionSettingsView: View {
     }
     @State var configuration: Configuration
     @Environment(\.dismiss) var dismiss
-    @StateObject var viewModel = SubscriptionSettingsViewModel()
```

Comment:
Can we leave out this rename. This view model is specific to this view so I have the feeling that adding some naming (that especially with two cases of combination for `subscription` and `settings`) can cause confusion.

Reply:
Yep, I think we can undo it now that this is no longer being injected in, happy to keep it as-is.

Reply:
Done ✅